docsrc/installation.rst

```diff
@@ -189,7 +192,7 @@ named ``.cmdstan``.  This directory will be created by the install script.
 
 .. code-block:: bash
 
-    python -m cmdstanpy.install_cmdstan
+    install_cmdstan --compiler True
```

Comment:
```suggestion
    install_cmdstan --compiler
```

The flag doesn't take any arguments, it is similar to `--progress` where passing it is equivalent to `compiler=True` and not passing (default) is False

Reply:
fixed